lib/internal/process/promises.js

```diff
@@ -44,6 +47,14 @@ function setupPromises(scheduleMicrotasks) {
         if (!process.emit('unhandledRejection', reason, promise)) {
           // Nobody is listening.
           // TODO(petkaantonov) Take some default action, see #830
+
+          if (traceUnhandledRejection && reason.stack) {
```

Comment:
`reason` can be of any type, including `null`. How abount printing `reason.stack` when `reason && reason.stack` and otherwise just `reason` itself?